app/controllers/spree/admin/users_controller.rb

```diff
@@ -46,7 +46,12 @@ def update
             @user.spree_roles = roles.reject(&:blank?).collect{ |r| Spree::Role.find(r) }
           end
 
-          flash.now[:success] = Spree.t(:account_updated)
+          message = if params[:user][:email] != @user.email
```

Comment:
Hi @jasonk357 thanks for the PR :smiley: 

These lines are good logic-wise, but could be a bit clearer for future devs trying to understand the code. It might be nice to extract this conditional logic into a human-readable (private) method. For example:

```ruby
     ...

     message = if new_email_unconfirmed?
          Spree.t(:email_updated)
        else
          Spree.t(:account_updated)
        end
     flash.now[:success] = message
  end
  render :edit
end

...

private

def new_email_unconfirmed?
   params[:user][:email] != @user.email
end
```

Reply:
c46fe7b